tests/end2end/test_single_operator_with_cntk_backend.py

```diff
@@ -17,6 +18,8 @@
 from keras.layers import Input, Dense, Conv2D, MaxPooling2D, AveragePooling2D, Conv2DTranspose, \
     Dot, Embedding, BatchNormalization, GRU, Activation, PReLU, LeakyReLU, ThresholdedReLU, Maximum, \
     Add, Average, Multiply, Concatenate, UpSampling2D, Flatten, RepeatVector, Reshape, Dropout
+if StrictVersion(keras.__version__) >= StrictVersion('2.1.5'):
```

Comment:
Can you check keras code history to see what's exact version when SeperableConv was introduced.

Reply:
2.1.3 for SeparableConv1D, 1.0.6 for 2D which is earlier than our specified version, code updated